corehq/apps/translations/validator.py

```diff
@@ -36,7 +36,9 @@ def __init__(self, app, uploaded_workbook, lang_to_compare, lang_prefix='default
         self.expected_rows = None
         self.lang_prefix = lang_prefix
         self.lang_cols_to_compare = [self.lang_prefix + self.app.default_language]
-        if lang_to_compare != self.app.default_language:
+        if lang_to_compare not in app.langs:
```

Comment:
Setting this to None is a bit confusing. I think this is equivalent and easier to read:
```
if lang_to_compare in app.langs and lang_to_compare != self.app.default_language:
    self.lang_cols_to_compare.append(self.lang_prefix + lang_to_compare)
self.default_language_column = self.lang_prefix + self.app.default_language
target_lang = lang_to_compare if lang_to_compare in app.langs else self.app.default_language
```

Reply:
Thank you @orangejenny. Updated